### ISSUE TITLE
command not found: antibody

### ISSUE THREAD

**Issue by tomspeak** [getantibody/antibody#150]
Hi,

I am having this odd issue that I can quite figure out, when my terminal loads, I get these errors:

```bash
/Users/tomspeak/dotfiles/antibody/antibody.init:1: command not found: antibody
/Users/tomspeak/dotfiles/antibody/antibody.init:2: command not found: antibody
```

But, if I then manually run `zsh` everything loads as expected.

`dotfiles/antibody/antibody.init`

```bash
source <(antibody init)
antibody bundle < "$DOTFILES/antibody/bundles"
```

`dotfiles/antibody/bundles`
```bash
zsh-users/zsh-completions
zsh-users/zsh-syntax-highlighting
frmendes/geometry
```

`.zshrc, lines 1:4`
```bash
export DOTFILES="$HOME/dotfiles"
export ZSH=$HOME/.oh-my-zsh

source $DOTFILES/antibody/antibody.init
```

I am running MacOS Sierra, with iTerm 2. I installed via `brew install antibody`.

I assume I am not loading antibody in the correct order, but I have followed the docs and other peoples dotfiles, not sure where I have gone wrong. Thanks!

My full dotfiles can be found here: https://github.com/tomspeak/dotfiles

**Comment by caarlos0** [comment]
Hi @tomspeak , is `/usr/local/bin` in your `$PATH` at the time `dotfiles/antibody/antibody.init` is called?

**Comment by tomspeak** [comment]
Bah, great catch. Thanks a lot, I moved the export up above my `antibody.init` and it's working flawlessly.

Funny note, in Terminal.app, it didn't complain at all! 

Thank you for the project. Finally all my zsh plugins and themes work and in lightning fast time, stumbled across this from https://github.com/frmendes/geometry, just as I started learning Go too, looking forward to dig into the source code.
